docs/source/details/mpi.rst

```diff
@@ -0,0 +1,51 @@
+.. _details-mpi:
+
+MPI Behavior
+============
+
+openPMD-api can be used within a MPI-parallel environment.
+The following API contracts need to be followed.
+
+Collective
+----------
+
+Series
+^^^^^^
+
+Generally, opening and closing the ``Series`` object with a MPI communicator is a collective operation.
+The communicator will be duplicated via ``MPI_Comm_dup`` in its constructor and freed in the destructor.
+
+The first access to ``Series::iterations::operator[]`` as well as ``Iteration::meshes::operator[]`` and ``Iteration::particles::operator[]`` are collective.
+
+Flush
+^^^^^
+
+Flush should be treated as collective operation.
```

Comment:
Besides bugs like #490 and #484, I think this is a design bug.
`flush()` should only ensure that user-provided buffers can be manipulated or freed again, if still hold by the user. We should make no assumptions about when something is written to disk.